subsys/app_event_manager/aem.ld

```diff
@@ -1,6 +1,6 @@
 ITERABLE_SECTION_ROM(event_type, 4)
 ITERABLE_SECTION_ROM(event_listener, 4)
-ITERABLE_SECTION_ROM(event_manager_postinit_hook, 4)
+ITERABLE_SECTION_ROM(app_event_manager_postinit_hook, 4)
 ITERABLE_SECTION_ROM(event_submit_hook, 4)
```

Comment:
Should we update other section names aswell? Or we leave internal definitions without app prefix ?